kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java

```diff
@@ -181,6 +181,10 @@
     FormLabel labelPing;
     @UiField
     FormLabel labelIgnore;
+    @UiField
```

Comment:
Add Sterwen to the copyright headers as contributor

Reply:
OK